### PR TITLE
Add double check deployments

### PR DESCRIPTION
**Pull request by MElborolossy**
### Description

Add double check deployment and fix related issues

### Changes

- Parameterize all tests with single and double-check deployments.
- Check that traefik and resources are ready before deployments.
- Replace assert on failed certificate with a warning.
- Update test docs with the latest.
- Make kubeapps deploy one deployment only #2513.
- Make Monitoring Stack deploy once after the last update #2551.

### Related Issues
#2201
#2513
#2551

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

**Comment by MElborolossy** [comment]
The latest state of the tests on deployed platinum vdc _deployed before the test_
## SIngle deployment 
All tests passed taking into consideration the following:
- Monitoring skipped related to #2538 
- Extend cluster fail _Wallet needs to be updated with the current VDC wallet_
image

## Double deployment
All tests passed taking into consideration the following:
- Monitoring skipped related to #2538 
- Taiga skipped related to #2296
- Extend cluster failed because  _Wallet needs to be updated with the current VDC wallet_
image

**Comment by MElborolossy** [comment]
Update the state of skipped and failed tests that passed
- Extend Cluster
image

- Single Deployment for monitoring stack
image

- Double Deployment for monitoring stack
image

**Comment by MElborolossy** [comment]
Taiga Also Passed in double deployment after this [update](https://github.com/threefoldtech/js-sdk/issues/2523#issuecomment-778776466)
image

File: tests/sals/automated_chatflows/test_vdc_dashboard.py [hunk 1]
```diff
@@ -4,26 +4,55 @@
...
+        traefik_expiry = j.data.time.now().timestamp + 300
+        while j.data.time.now().timestamp < traefik_expiry and not is_traefik_ready:
+            rc, system_pods, err = cls.kube_manager._execute(
+                f"kubectl get pods --kubeconfig {cls.kube_manager.config_path} --namespace kube-system | grep traefik"
+            )
+            running = system_pods.count("Running") + system_pods.count("Completed")
+            all = len(system_pods.splitlines())
+            if running == all:
+                cls.info("Traefik is ready")
+                is_traefik_ready = True
```
Comment (AhmedHanafy725) [inline tests/sals/automated_chatflows/test_vdc_dashboard.py:54]: I think no need for this variable, if traefik is running, you can just break

Comment (MElborolossy) [reply tests/sals/automated_chatflows/test_vdc_dashboard.py:54]: Add assert on it will be better.

**Review by waleedhammam — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**